wrapper/src/test/build.gradle.kts

```diff
@@ -42,6 +42,7 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-jdbc:2.7.13") // 2.7.13 is the last version compatible with Java 8
     testImplementation("org.mockito:mockito-inline:4.11.0") // 4.11.0 is the last version compatible with Java 8
     testImplementation("software.amazon.awssdk:rds:2.20.49")
+    testImplementation("software.amazon.awssdk:sts:2.20.49")
```

Comment:
NIT: let's put sdk dependencies in alphabetical order

Reply:
Reordered